models/models.go

```diff
@@ -0,0 +1,49 @@
+package models
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+// CodeProcessRequest represents the payload sent to the code runner lambda
+type CodeProcessRequest struct {
+	Code     string `json:"code"`
+	Language string `json:"language"`
+}
+
+// CodeProcessResponse represents the payload returned by the code runner lambda
+type CodeProcessResponse struct {
+	Output   string `json:"output"`
+	ExitCode int    `json:"exitCode"`
+}
+
+// LanguageProperties represents properties for running each supported language
+type LanguageProperties struct {
+	Name           string `json:"langName"`
+	Extension      string `json:"extension"`
+	Placeholder    string `json:"placeholder"`
+	FileName       string `json:"fileName"`
+	RunCommand     string `json:"runCmd"`
+	CompileCommand string `json:"compileCmd"`
+}
+
+// LanguageConfig represents the model matching the languages.json file
+type LanguageConfig map[string]LanguageProperties
+
+// ParseLanguageConfig parses the languages file into a LanguageConfig model
+func ParseLanguageConfig() (LanguageConfig, error) {
+	data, err := ioutil.ReadFile("../../languages.json")

```

Comment:
will be cooler if the path is a param to this parse func

Reply:
updated